common/local-settings.ts

```diff
@@ -17,6 +17,7 @@ export interface ShieldBatteryAppSettingsData {
 
 export interface LocalSettingsData extends ShieldBatteryAppSettingsData {
   runAppAtSystemStart: boolean
+  startAppMinimized: boolean
```

Comment:
I would rather name this setting something like `runAppAtSystemStartMinimized` to make it clear it's only applied to the `runAppAtSystemStart` setting, even if it's a bit of a mouthful :d